hydromt_fiat/workflows/equity_data.py

```diff
@@ -186,6 +185,11 @@ def download_shp_geom(self, year_data: int, counties: List[str]):
 
         #TODO Save final file as geopackage
         self.block_groups = gpd.GeoDataFrame(pd.concat(block_groups_list))
+        
+        # Create string from GEO_ID short 
+        for index, row,in self.block_groups.iterrows():
+            BG_string = f"BG: {row['GEOID_short']}"
+            self.block_groups.at[index, 'GEOID_short'] = BG_string
```

Comment:
this is the same as the method rename_geoid_short, so why repeat and not use the method?

Reply:
done, moved it to utils.py :)